### PR TITLE
[test](fix) Case bug

### PR DESCRIPTION
**Pull request by shuke987**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: regression-test/suites/query_p0/aggregate/push_filter_through_agg.groovy [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 suite("push_filter_through_agg") {
     sql """DROP TABLE IF EXISTS t_push_filter_through_agg"""
+    sql """DROP VIEW IF EXISTS view_i"""
```
Comment (englefly) [inline regression-test/suites/query_p0/aggregate/push_filter_through_agg.groovy:20]: "create view XXX if exists" is better

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by englefly — APPROVED** [review]

File: regression-test/suites/query_p0/aggregate/push_filter_through_agg.groovy (hunk 1 above)
Comment (shuke987) [reply regression-test/suites/query_p0/aggregate/push_filter_through_agg.groovy:20]: create view XXX if exists may not handle the situation that the structure of the table changed by somebody

**Pull request merged**